orchagent/port.h

```diff
@@ -47,6 +49,15 @@ class Port
         UNKNOWN
     } ;
 
+    enum Dependency {
+	    ACL,
+	    LAN,
+	    INTF,
+	    FDB,
+	    QOS,
+	    MIRROR,
+    };
+
```

Comment:
This is the list of dependency check?  
Reference counts and the function for setting the bits will be added later?